app/src/androidTest/java/com/okta/idx/android/SelfServiceRegistrationTestEndToEnd.kt

```diff
@@ -49,6 +51,52 @@ import org.junit.runner.RunWith
 class SelfServiceRegistrationTestEndToEnd {
     @get:Rule val activityRule = ActivityScenarioRule(MainActivity::class.java)
 
+    private var profile: A18NProfile? = null
+
+    @After
+    fun deleteProfile() {
+        profile?.let {
+            A18NWrapper.deleteProfile(it)
+        }
+    }
+
+    // Mary signs up for an account with Password, sets up required Email factor, then skips
+    // optional SMS.
+    @Test fun scenario_4_1_1() {
+        profile = A18NWrapper.createProfile()
+        activityRule.scenario.moveToState(Lifecycle.State.RESUMED)
+        onView(withId(R.id.self_service_registration_button)).perform(click())
+        waitForElement(FIRST_NAME_EDIT_TEXT)
+
+        onView(withId(R.id.first_name_edit_text)).perform(replaceText("Mary"))
```

Comment:
This is going to flood our test org with users. Please leave this open until I have my PR for user management so we can delete these users.